ui-admin/src/portal/publish/PortalEnvDiffProvider.tsx

```diff
@@ -33,7 +32,8 @@ const PortalEnvDiffProvider = ({ portal, studyShortcode, updatePortal }: PortalE
       const envIndex = updatedPortal.portalEnvironments.findIndex(env => env.environmentName === result.environmentName)
       updatedPortal.portalEnvironments[envIndex] = result
       updatePortal(updatedPortal)
-      navigate(studyPublishingPath(portal.shortcode, studyShortcode))
+      // for now, do a hard reload to make sure all changes are propagated
+      window.location.pathname = studyPublishingPath(portal.shortcode, studyShortcode)
```

Comment:
I know the focus is to unblock, but in the interest of not accumulating small TODOs, it would only be a few line change to do an `await portalContext.reloadPortal(portal.shortcode)` above and keep the `navigate` here.